packages/server/graphql/intranetSchema/README.md

```diff
@@ -0,0 +1,11 @@
+## Intranet/Admin interface 
+
+This part of Parabol's API is available only for admins. To become an admin user has to have `su` role set.
+
+`su` role can be set via RethinkDB’s data explorer, on local environment it’s available at: http://localhost:8080/#dataexplorer
+```
+r.db('actionDevelopment').table('User').filter({email: 'set-your-email-here'}).update({rol:'su'})
```

Comment:
could you add the SQL here, too? want to keep the User data across both DBs aligned

Reply:
🙈  I forgot about the PG, and sure, added!